core/include/detray/geometry/surface.hpp

```diff
@@ -0,0 +1,145 @@
+/** Detray library, part of the ACTS project (R&D line)
```

Comment:
Could you add this file to `CMakeLists.txt`?

Reply:
Right

Reply:
While I am at it, I also cleaned out a number of other old headers